frontend/src/app/leaderboard/topStats.tsx

```
@@ -0,0 +1,64 @@
+              ></path>
+            </svg>
+          </div>
+          <div className="stat-title text-xs md:text-base">Total Rewards</div>
+          <div className="stat-value  text-base md:text-3xl  text-secondary">1000</div>
+        </div>
+
+        <div className="stat">
+          <div className="stat-figure hidden md:block text-secondary">
+            <div className="avatar online">
+              <div className="w-8 md:w-16 rounded-full">
+                <img src="https://img.daisyui.com/images/stock/photo-1534528741775-53994a69daeb.webp" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)